src/erlcloud_ec2.erl

```diff
@@ -1188,6 +1188,7 @@ extract_image(Node) ->
      {description, get_text("description", Node)},
      {root_device_type, get_text("rootDeviceType", Node)},
      {root_device_name, get_text("rootDeviceName", Node)},
+     {creation_date, erlcloud_xml:get_time("creationDate", Node)},
```

Comment:
you're right. this fork does not have that test at all.
here is an example - https://github.com/alertlogic/erlcloud/blob/master/test/erlcloud_ec2_tests.erl#L430-L475